_includes/header.html

```diff
@@ -10,7 +10,7 @@ <h1 class="logo">{{ site.title }}</h1>
       <div id="navBtn">menu</div>
       <ul id="nav" class="dropdown__menu hide">
         {% for link in site.header_themes %}
-          <li><a href="{{link.url}}">{{link.text}}</a></li>
+          <li><a href="{{link.url}}">{{ site.translations[site.lang].pages[link.text] }}</a></li>
```

Comment:
This is suboptimal, but:

* We cannot use `{% t pages.foo %}` since we need to access the attribute of `pages` dynamically.
* I tried `{% t pages[link.text] %}` but apparently the `var[]` syntax does not work in this context.

Reply:
I wonder whether `t "pages.foo"` would have worked. in theory `t` is just a function, and the key of `"pages.foo"` is the key, but maybe the translate function is getting confused because it's in a loop :shrug:. either way, approving because it works :-D

Reply:
I thought that might work and tries `t "pages" + link.text`, but then realized I should just read the [Liquid docs](https://shopify.github.io/liquid/filters/). :) Current solution looks a bit nicer.